lib/fluent/plugin_helper/event_emitter.rb

```diff
@@ -47,7 +47,11 @@ def event_emitter_used_actually?
 
       def event_emitter_router(label_name)
         if label_name
-          Engine.root_agent.find_label(label_name).event_router
+          if label_name == ""
```

Comment:
> This patch adds a way in event_emitter_router(label_name) to get the root event router. To use an empty string "" for the default router is just a proposal, maybe @DEFAULT would fit better.

I guess `@DEFAULT` is better.
I don't want to use empty string: `""` as default router.
I don't like to implicit meaning literals.

Reply:
I'm not sure `@DEFAULT` is an appropriate name.
`@ROOT` might be more suitable because:

> get the root event router.

Reply:
label_name cannot be empty string(`""`).
Due to: https://github.com/fluent/fluentd/blob/master/lib/fluent/root_agent.rb#L133